packages/apps/graph/src/graph/objects/Transaction.ts

```diff
@@ -8,22 +8,30 @@ export default builder.prismaNode('Transaction', {
     badResult: t.string({
       nullable: true,
       resolve({ badResult }) {
-        return !badResult ? undefined : JSON.stringify(badResult);
+        return badResult === undefined || badResult === null || badResult === ''
```

Comment:
Since this check is done multiple times, maybe creating a utils function that would check this?

Reply:
Agreed 👍